lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1830,24 +1841,91 @@ namespace Js
             bytecodeGen = HeapNew(Wasm::WasmBytecodeGenerator, this, *ppSourceInfo, reader);
             wasmModule = bytecodeGen->GenerateModule();
 
-            Wasm::WasmFunction ** functionArray = wasmModule->functions->GetBuffer();
+            Wasm::WasmFunction ** functionArray = wasmModule->functions;
 
             Var* moduleMemoryPtr = RecyclerNewArrayZ(GetRecycler(), Var, wasmModule->memSize);
 
             Var* heap = moduleMemoryPtr + wasmModule->heapOffset;
 
             exportObj = JavascriptOperators::NewJavascriptObjectNoArg(this);
-            Js::Var exportsNamespace = JavascriptOperators::NewJavascriptObjectNoArg(this);
 
-            PropertyRecord const * exportsPropertyRecord = nullptr;
+            Var* localModuleFunctions = moduleMemoryPtr + wasmModule->funcOffset;
 
+            FrameDisplay * frameDisplay = RecyclerNewPlus(GetRecycler(), sizeof(void*), FrameDisplay, 1);
```

Comment:
I agree, there will be some effort in the near future to cleanup the current code base of web assembly branch before we move on to the next work items